main.js

```diff
@@ -36,6 +36,7 @@ const sockets = combine(
   require('patchbay-book'),
   // require('patch-network),
   require('patch-settings'), // might need to be in patchbay
+  require('patch-drafts'),
```

Comment:
[minor] : given how woven in this module is, it needs to be part of the patchbay set of modules up on line 9